src/libs/OptionsListUtils.js

```diff
@@ -25,6 +25,19 @@ let preferredLocale;
 // will update when the OptionsListUtils re-loads. But will stay the same color for the life of the JS session.
 const defaultAvatarForUserToInvite = getDefaultAvatar();
 
+/**
+ * Adds expensify SMS domain (@expensify.sms) if login is a phone number and if it's not included yet
+ *
+ * @param {String} login
+ * @return {String}
+ */
+function addSMSDomainIfPhoneNumber(login) {
+    if (Str.isValidPhone(login) && !Str.isValidEmail(login)) {
+        return login + CONST.SMS.DOMAIN;
+    }
+    return login;
```

Comment:
We might prefer to return early, but NAB

Reply:
Or even make it a ternary, but I'm not sure either is more legible than this. I'd say let's keep it like it is now.